gnc/sub8_perception/nodes/buoy_2d.py

```diff
@@ -166,44 +173,36 @@ def get_biggest(self, contours):
             return None
 
     def find_single_buoy(self, img, buoy_type):
-        assert buoy_type in self.buoys[buoy_type], "Buoys_2d does not know buoy color: {}".format(buoy_type)
-        ncc_channel = 2
-        hsv = cv2.cvtColor(img, cv2.COLOR_BGR2HSV)
-
-        color_mask, _range = threshold_tools.param_thresh(img, self.buoys[buoy_type])
-        bgr_range = np.average(_range, axis=1)
-
-        hsv_vect = threshold_tools.bgr_vec_to_hsv(bgr_range)
+        if buoy_type != 'yellow':
```

Comment:
(I am ok with delaying fixing this important problem because we need to spend some time on infrastructure for storing big classifiers in order to make this whole boosting thing work)